Bio/Seq.py

```diff
@@ -1708,6 +1708,26 @@ def __init__(self, data):
         else:
             self._data = data  # assumes the input is an array
 
+    @property
+    def data(self):
+        warnings.warn(
+            "Accessing MutableSeq.data has been deprecated, as it is a private "
+            "attribute. Please use indexing to access the sequence contents of "
+            "a MutableSeq object.",
+            BiopythonDeprecationWarning,
+        )
+        return self._data
+
+    @data.setter
+    def data(self, value):
+        warnings.warn(
+            "Accessing MutableSeq.data has been deprecated, as it is a private "
+            "attribute. Please use indexing to access the sequence contents of "
+            "a MutableSeq object.",
+            BiopythonDeprecationWarning,
+        )
+        self._data = value
+
```

Comment:
Maybe a slight rewording, "is now a private attribute"?

Reply:
done